app/views/services/edit.html.erb

```diff
@@ -20,13 +20,13 @@
     <% service.pages.each do |page| %>
       <section class="form-step" aria-label="<%= t('aria.' + page.type) %>">
 
-        <%= flow_thumbnail(page, service.service_id) %>
+        <%= flow_thumbnail(page) %>
 
         <ul class="govuk-navigation component-activated-menu"
             data-activator-text="<%= t('pages.actions') %>"
             data-activator-classname="form-step_button"
             data-component="PageActionMenu"
-            data-page-heading="<%= flow_thumbnail_heading(page) %>"
+            data-page-heading="<%= page.title %>"
```

Comment:
Are you sure this will work?
I think the reason for the flow_thumbnail_heading(page) output was due to a page.multiple and page.single requiring different 'title' values. For a single question page it was page.title but for a multiple question page it was page.first_component.title (not actual code just illustration of value required).

Reply:
I had to do a fix in the presenter:

https://github.com/ministryofjustice/fb-metadata-presenter/pull/176

So the behaviour will now match the same behaviour as before